src/Symfony/Component/Validator/Resources/translations/validators.id.xlf

```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>Bidang ini tidak diharapkan.</target>
```

Comment:
hi @javiereguiluz i use the official translation list from government.
you can view here http://www.perpusnas.go.id/assets/uploads/2016/10/Panduan_Pembakuan_Istilah.pdf
the list from national library of Indonesia.